packages/openid4vc-holder/src/presentations/selection/PexCredentialSelection.ts

```diff
@@ -0,0 +1,334 @@
+import type { PresentationSubmission, PresentationSubmissionRequirement, SubmissionEntry } from './types'
```

Comment:
I don't think there is anythinng we can do about this, but this file contains quite some complex code (and quite some TODO's). Would be nice if we can clean this up later on.

Reply:
Everything related to vp/siop is not yet finished. That explains the TODO's and some issues you mentioned in this review. Just the vci holder side is 'done'. So I will clean this up in the future.